src/utilities/pathnames.js

```diff
@@ -13,6 +13,11 @@ const pathnames = {
     path: '/workspaces/*',
     title: 'Workspaces',
   },
+  'workspace-detail': {
+    link: '/workspaces/detail/:workspaceId',
+    path: '/workspaces/detail/:workspaceId/*',
```

Comment:
No need for detail in path
```suggestion
    link: '/workspaces/:workspaceId',
    path: '/workspaces/:workspaceId/*',
```

Reply:
Fixed